libs/cardano-ledger-test/src/Test/Cardano/Ledger/Constrained/TypeRep.hs

```diff
@@ -315,6 +315,10 @@ data Rep era t where
   NumDormantEpochsR :: Era era => Rep era EpochNo
   CommitteeStateR :: Era era => Rep era (CommitteeState era)
   VStateR :: Era era => Rep era (VState era)
+  PrevPParamUpdateChildrenR :: Era era => Rep era (GovActionId (EraCrypto era)) -> Rep era (Set (GovActionId (EraCrypto era)))
+  PrevHardForkChildrenR :: Era era => Rep era (GovActionId (EraCrypto era)) -> Rep era (Set (GovActionId (EraCrypto era)))
+  PrevCommitteeChildrenR :: Era era => Rep era (GovActionId (EraCrypto era)) -> Rep era (Set (GovActionId (EraCrypto era)))
+  PrevConstitutionChildrenR :: Era era => Rep era (GovActionId (EraCrypto era)) -> Rep era (Set (GovActionId (EraCrypto era)))
```

Comment:
You'll need to ask @TimSheard about correctness of this, I don't have the full grasp on this framework yet

Reply:
@TimSheard, am I doing the right thing here in `cardano-ledger-test`?